.travis.yml

```diff
@@ -23,4 +23,4 @@ script:
     # t1004.17 needs "git merge-recursive" (which also isn't documented)
     # t1014.26 needs "git config --unset-all"
     # t3000.7 requires git pack-refs
-    - GIT_SKIP_TESTS='t0008.30[5-9] t0008.310 t0008.321 t0008.323 t0008.37[0-9] t0008.38[0-7] t0008.39[1-2] t1004.1[6-7] t1014.26 t3000.7 t2018.[6-7] t2018.[9] t2018.1[5-8]' ./official-git/run-tests.sh t0000-basic.sh t0004-unwritable.sh t0007-git-var.sh t0008-ignores.sh t0010-racy-git.sh t0062-revision-walking.sh t0070-fundamental.sh t0081-line-buffer.sh t1000-read-tree-m-3way.sh t1001-read-tree-m-2way.sh t1002-read-tree-m-u-2way.sh t1003-read-tree-prefix.sh t1004-read-tree-m-u-wf.sh t1005-read-tree-reset.sh t1008-read-tree-overlay.sh t1009-read-tree-new-index.sh t1011-read-tree-sparse-checkout.sh t1012-read-tree-df.sh t1014-read-tree-confusing.sh t1100-commit-tree-options.sh t1304-default-acl.sh t1403-show-ref.sh t2000-checkout-cache-clash.sh t2001-checkout-cache-clash.sh t2002-checkout-cache-u.sh t2006-checkout-index-basic.sh t2009-checkout-statinfo.sh t2014-switch.sh t2018-checkout-branch.sh t2100-update-cache-badpath.sh t3000-ls-files-others.sh t3002-ls-files-dashpath.sh t3006-ls-files-long.sh t3020-ls-files-error-unmatch.sh t3800-mktag.sh t4113-apply-ending.sh t4123-apply-shrink.sh t7062-wtstatus-ignorecase.sh t7511-status-index.sh
```

Comment:
Due to recent efforts we can stop skipping 6 ignore tests.

Reply:
I wonder if we can split this over multiple lines with `\` to get better diffs here, it's always hard to tell what's added and what's removed..

Reply:
is this t0008.305 through t0008.310 being added?

Reply:
Yes, that's right. I'm not a yaml expert. Do multi-line strings work that way here?

Reply:
These might work: https://yaml-multiline.info

I can give it a try sometime.

Reply:
If you want to revert so that it passes we can figure the spacing out later

Reply:
I pushed the list of ignores and the list of test suites into the shell script where it can be run in development environment and in travis environment more easily.